### PR TITLE
Batch dependabot updates

### PR DESCRIPTION
**Pull request by baruchiro**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-- package-ecosystem: npm
-  directory: "/nuxt/"
-  open-pull-requests-limit: 0
-  schedule:
-    interval: "weekly"
+  - package-ecosystem: npm
+    directory: "/nuxt"
+    open-pull-requests-limit: 0
+    schedule:
+      interval: "weekly"
```
Comment (baruchiro) [inline .github/dependabot.yml:19]: I hope it will stop to suggest upgrades for `nuxt`

File: .github/dependabot.yml [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
-  ignore:
-  - dependency-name: electron
-  - dependency-name: spectron
-  - dependency-name: typescript
-  - dependency-name: sass-loader
-    versions: ["11.x"]
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```
Comment (baruchiro) [inline .github/dependabot.yml:6]: It create a lot of noises

File: .github/dependabot.yml (hunk 1 above)
Comment (brafdlog) [reply .github/dependabot.yml:19]: Why would it stop from this change? If you don't want it to udpate nuxt, why not remove this whole chunk?
```suggestion
```

Comment (baruchiro) [reply .github/dependabot.yml:19]: You know, I thought about that. It is OK for me, worst case we can tag the last commit with `nuxt` to easily restore it if we will want to.
I will do that in a different PR, let's merge this PR as is.

**Review by brafdlog — APPROVED** [review]

**Pull request merged**